src/coinjoin/client.cpp

```diff
@@ -967,11 +966,15 @@ bool CCoinJoinClientManager::DoAutomaticDenominating(ChainstateManager& chainman
     // If we've used 90% of the Masternode list then drop the oldest first ~30%
     int nThreshold_high = nMnCountEnabled * 0.9;
     int nThreshold_low = nThreshold_high * 0.7;
-    WalletCJLogPrint(m_wallet, "Checking vecMasternodesUsed: size: %d, threshold: %d\n", (int)vecMasternodesUsed.size(), nThreshold_high);
+    size_t nUsedMasternodes{m_mn_metaman.GetUsedMasternodesCount()};
 
-    if ((int)vecMasternodesUsed.size() > nThreshold_high) {
-        vecMasternodesUsed.erase(vecMasternodesUsed.begin(), vecMasternodesUsed.begin() + vecMasternodesUsed.size() - nThreshold_low);
-        WalletCJLogPrint(m_wallet, "  vecMasternodesUsed: new size: %d, threshold: %d\n", (int)vecMasternodesUsed.size(), nThreshold_high);
+    WalletCJLogPrint(m_wallet, "Checking nUsedMasternodes: %d, threshold: %d\n", (int)nUsedMasternodes, nThreshold_high);
```

Comment:
nit(code-style):

`nUsedMasternodes` is not matched with our code style.

prefix `n` should not be used.
case of variable should be snake-case, such as `used_masternodes` or `used_masternodes_count` or something similar.

See doc/developer-notes.md:
>   - Variable (including function arguments) and namespace names are all lowercase and may use `_` to
    separate words (snake_case).



I'd suggest to change log record to be a bit more human rather than use variable name in logs (even if it's debug):

```suggestion
    WalletCJLogPrint(m_wallet, "Checking threshold of used masternodes for coinjoin: amount=%d, threshold=%d\n", (int)nUsedMasternodes, nThreshold_high);
```